commands/users/unbanUserIdInMessage.js

```diff
@@ -0,0 +1,62 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { guildId, BATId, BALId, BAGId, BADId } = require('../../config.json');
+const { ChannelType } = require('discord-api-types/v9');
+const { MessageAttachment } = require('discord.js');
+
+const ALL_GUILD_IDS = [guildId, BATId, BALId, BAGId, BADId];
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('unbanuseridinmessage')
+        .setDescription('Gets a list of user ids from a message(s) and unbans them from all servers.')
+        .addStringOption(option =>
+            option.setName('message_ids')
+                .setDescription('The message id(s) you would like to get the user ids from')
+                .setRequired(true))
+        .addChannelOption(option =>
+            option.setName('channel')
+                .setDescription('The channel the message(s) are in')
+                .addChannelTypes([ChannelType.GuildText, ChannelType.GuildPublicThread, ChannelType.GuildPrivateThread])
+                .setRequired(true))
+        .setDefaultPermission(false),
+    async execute(interaction) {
+        const options = interaction.options;
+        const messageIds = options.getString('message_ids');
+        const channel = options.getChannel('channel');
+
+        await interaction.deferReply();
+
+        const unbanUsersOnServers = async (messageId) => {
+            channel.messages.fetch(messageId).then(async (msg) => {
+                let membersUnbanned = '';
+                const userIds = msg.content.split(' ');
```

Comment:
wondering if we should use a csv of user ids at a file_url option like in `addRole,js` and then leverage our common `openFileAndDo` for this because a message content could be anything and it would be hard to ensure we are actually getting user ids in the message content 😢  
```
 DiscordUtil.openFileAndDo(attachmentURL, unbanUsersOnServers, interaction);
 ```

Reply:
sounds like a good idea, i will look into this <3

Reply:
i wasnt able to use the `openFileAndDo` function because passing the callback for `guild.members.kick( )` and  `guild.members.ban( )` was throwing an error 😞